src/protocolgame.cpp

```diff
@@ -1484,7 +1484,15 @@ void ProtocolGame::sendContainer(uint8_t cid, const Container* container, bool h
 		msg.addString("Browse Field");
 	} else {
 		msg.addItem(container);
-		msg.addString(container->getName());
```

Comment:
```
const std::string& containerName = container->getName();
msg.addString(containerName.empty() ? "item of type" + container->getID() : containerName);
```

Reply:
what about this?
`msg.addString(containerName.empty() ? "item of type" + container->getID() : container->getName());`

Reply:
I don't think so
![obraz](https://user-images.githubusercontent.com/5205079/80918679-81acfd80-8d66-11ea-8053-cdb03a850c5f.png)

Reply:
Shouldn't this be built into `container->getName()` itself?

Reply:
@DSpeichert this is absolute awesome idea

Reply:
getName() method added